src/net_processing.cpp

```diff
@@ -1162,7 +1162,7 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
     }
 }
 
-uint32_t GetFetchFlags(CNode* pfrom, const CBlockIndex* pprev, const Consensus::Params& chainparams) {
+uint32_t GetFetchFlags(CNode* pfrom) {
```

Comment:
Before `GetFetchFlags` was using `IsWitnessEnabled`, not `fHaveWitness`and `GetLocalServices`